README.md

```diff
@@ -50,3 +50,40 @@ This repo contains a [`flake.nix`](flake.nix) file which you can use for example
 ```shell
 nix run github:brumhard/krewfile# -- -help
 ```
+
+**Using home-manager module**
+
+If you using flakes to manage your system, add the `krewfile` input as follows:
+
+```nix
+# ...
+inputs.krewfile = {
+  url = "github:brumhard/krewfile";
+  inputs.nixpkgs.follows = "nixpkgs";
+};
+```
+
+You can now access the module via `inputs.krewfile.homeManagerModules.krewfile`.
+Assuming a setup with properly configured imports via `home-manager.extraSpecialArgs`, the module can be imported and configured as shown below:
+
+```nix
+{ inputs, pkgs, ... }: {
+  imports [
+    # ...
+    inputs.krewfile.homeManagerModules.krewfile
+  ];
+
+  programs.krewfile = {
+    enable = true;
+    krewPackage = pkgs.krew;
+    plugins = [
+      "explore"
+      "modify-secret"
+      "neat"
+      "oidc-login"
+      "pv-migrate"
+      "stern"
+    ];
+  };
```

Comment:
```suggestion
  imports = [
    # ...
    inputs.krewfile.homeManagerModules.krewfile
  ];

  programs.krewfile = {
    enable = true;
    krewPackage = pkgs.krew;
    plugins = [
      "krew" # krew manages itself
    
      "explore"
      "modify-secret"
      "neat"
      "oidc-login"
      "pv-migrate"
      "stern"
    ];
  };
```

Reply:
That's what I found when trying it out 🙂

Reply:
Wait but is krew not managed through nix now? So if krew manages itself, this could break by a `flake-update`?

Reply:
Just tested again. With this setup we manage `krew`. If you do include `krew` in the plugins list you also get `kubectl-krew`, so you can run `k krew list` for example.
The issue currently with my setup is that I have krew installed as a plugin (which is not the default on a fresh install) and when you try to uninstall it, you get an error `failed to uninstall plugin krew: self-uninstall not allowed`.
So I think it doesn't really matter what the docs say, people will need to check their local setup anyways before using the module.

Reply:
Pls change `imports [` to `imports = [` tho

Reply:
I improved the error output in https://github.com/brumhard/krewfile/pull/6 so it should be easy to find the problem